src/EditorFeatures/TestUtilities/Roslyn.Services.Test.Utilities.csproj

```diff
@@ -7,36 +7,35 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.Test.Utilities</RootNamespace>
-    <AssemblyName>Roslyn.Services.Test.Utilities</AssemblyName>
```

Comment:
FYI it was somewhat unofficial but I think @Pilchie and others somewhat used "Roslyn"-prefixed assemblies to mean things that didn't ship. Might be worth calling out in an email that such a system is dead now. It was already 99% dead anyways, but not sure if anybody else still clinged to that.